packages/components/src/internal/components/entities/SingleParentEntityPanel.tsx

```diff
@@ -41,14 +41,24 @@ interface Props {
     onChangeParentValue?: (name: string, value: string | any[], index: number) => void;
     onInitialParentValue?: (value: string, selectedValues: List<any>, index: number) => void;
     onRemoveParentType?: (index: number) => void;
-    parentDataType: EntityDataType;
+    parentDataTypes: EntityDataType[];
```

Comment:
Passing `EntityDataType[]` is not necessary as `<SingleParentEntityPanel/>` only ever refers to the first instance in the array. I would revert this to be called `parentDataType` and just pass in the first one. Results in a lot less unnecessary changes to this component.

Reply:
Good catch. Fixed